src/main/java/io/r2dbc/postgresql/message/backend/ReadyForQuery.java

```diff
@@ -104,10 +103,10 @@ static ReadyForQuery decode(ByteBuf in) {
         }
 
         static TransactionStatus valueOf(byte b) {
-            return Arrays.stream(values())
-                .filter(type -> type.discriminator == b)
-                .findFirst()
-                .orElseThrow(() -> new IllegalArgumentException(String.format("%c is not a valid transaction status", b)));
+            if ( b == IDLE.discriminator ) return IDLE;
```

Comment:
Can you please match this to the style you used in `BackendMessageDecoder` instead?